microsoft/testsuites/storage/storagesuite.py

```diff
@@ -79,6 +100,75 @@ def verify_disk_with_nobarrier(self, node: Node, environment: Environment) -> No
         _stop_raid(node)
         _make_mount_raid(node, partition_disks)
 
+    @TestCaseMetadata(
+        description="""
+        This test case is to
+            1. Attach 2 512GB premium SSD disks
+            2. Create a 100GB partition for each disk using fdisk
+            3. Create a RAID type 1 device using partitions created in step 2
+            4. Run fio against raid0 with verify option for 100 times
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            disk=schema.DiskOptionSettings(
+                disk_type=schema.DiskType.PremiumSSDLRS,
+                data_disk_iops=search_space.IntRange(min=2300),
+                data_disk_count=search_space.IntRange(min=2),
+            ),
+        ),
+    )
+    def verify_disk_with_fio_verify_option(self, node: Node) -> None:
+        pattern = re.compile(
+            r"error=Invalid or incomplete multibyte or wide character", re.M
+        )
+        disk = node.features[Disk]
+        data_disks = disk.get_raw_data_disks()
+        disk_count = len(data_disks)
+        assert_that(disk_count).described_as(
+            "At least 2 data disk for testing."
+        ).is_greater_than(1)
+        partition_disks = _format_disk(node, data_disks, last_sector="+100G")
+        _stop_raid(node)
+        _make_mount_raid(
+            node,
+            partition_disks,
+            level="mirror",
```

Comment:
Is it the same as "0"? If so, it doesn't need to support both number and string.

Reply:
I thought they are the same, but actually only when set it as mirror, the issue can be repro.

Reply:
Add a comment to specify that, in case it's changed to "0" by someone in future.